task/runner.go

```diff
@@ -324,7 +341,9 @@ func (r *runner) scheduleTaskStep(ctx context.Context, taskID, step string, inpu
 }
 
 func (r *runner) publishTaskResult(ctx context.Context, task data.TaskInfo, output *data.TaskOutput, resultErr error) error {
-	resultErr = humanizeError(resultErr)
+	if r.HumanizeErrors {
```

Comment:
Just a curiosity, why is this flag needed? Don't we always want for the errors to be humanized?

Reply:
Not on staging! Will save a few debugging round trips to have the raw error there